catkin_tools/jobs/cmake.py

```diff
@@ -286,8 +286,7 @@ def create_cmake_build_job(context, package, package_path, dependencies, force_c
 
     # Pre-clean command
     if pre_clean:
-        make_args = handle_make_arguments(
-            context.make_args + context.catkin_make_args)
```

Comment:
Adding `context.catkin_make_args` here for `cmake` type jobs like in [catkin.py:464](https://github.com/catkin/catkin_tools/blob/613771f330c5b6ed86f3fec89f6e3214e21558e5/catkin_tools/jobs/catkin.py#L464-L467) was probably a bug.

But in which cases would it be useful to add more arguments to `make clean` at all? At least this breaks use cases like
```sh
catkin build --pre-clean --make-args tests
```